backend/src/middleware/helpers/email/sendMail.ts

```diff
@@ -44,6 +46,11 @@ if (!hasEmailConfig) {
         user: CONFIG.SMTP_USERNAME,
         pass: CONFIG.SMTP_PASSWORD,
       },
+      dkim: hasDKIMData && {
```

Comment:
Not totally sure about the case were `!hasDKIMData` but since it seem to work like this I would approve

Reply:
I tested it and the result is if `hasDKIMData` is `false` then the result is:

```js
transporterParameter:  { test: true, dkim: undefined }
```